src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsTableHeaderUI.java

```diff
@@ -89,6 +89,10 @@ public Component getTableCellRendererComponent(JTable table, Object value,
             this.isSelected = isSelected;
             this.hasFocus = hasFocus;
             this.column = column;
+            if(table != null)
```

Comment:
let's add space after `if`

Reply:
And put the opening brace on the same line as `if` itself to follow the Java coding style and the style that's used in below.